### PR TITLE
Fish-pepper updates and adding alpine jdk11 image

### PR DESCRIPTION
**Pull request by jukie**
Related to #36 
Bumps Jolokia to 1.6.2
Bumps alpine base to 3.10
Adds an alpine image for jdk11

### CODE REVIEW

File: images.yml [hunk 1]
```diff
@@ -56,7 +57,6 @@ config:
       fish-pepper:
         # Ignore for for Alpine and JBoss as there are no Java 11 packages
```
Comment (rhuss) [inline images.yml]:
```suggestion
        # Ignore for JBoss as there are no Java 11 packages
```

**Review by rhuss — COMMENTED** [review]
looks good, one minor comment, but ready to merge after applied.

**Comment by jukie** [comment]
Thanks, I’ll update this now.

**Review by rhuss — APPROVED** [review]
thanks

**Pull request merged**